.circleci/merge_base_branch.sh

```diff
@@ -9,13 +9,16 @@ source $BASH_ENV
 
 if [[ -z "${CIRCLE_PR_NUMBER}" ]]
 then
-    echo "Can't figure out what the PR number is!"
-    exit 1
+    echo "Can't figure out what the PR number is! Assuming develop."
```

Comment:
this looks unrelated, though useful.

I would s/Assuming develop/Assuming it will be based on develop/ though.